addons/website_sale/views/templates.xml

```diff
@@ -1393,7 +1393,7 @@
                     <th></th>
                     <th class="text-center td-qty">
                         <t t-if="show_qty">
-                            Quantity
+                            <span>Quantity</span>
                         </t>
```

Comment:
```xml
                        <span t-if="show_qty">
                            Quantity
                        </span>
```
Note that this would work too regarding edition capability, don't change it tho, it's less stable friendly as it changes XML structure (adding a new one is fine-ish). Just mentioning this as if we think about it in master forward port, it's worth changing.